app/config/config_prod.yml

```diff
@@ -83,7 +83,7 @@ monolog:
         timing:
             type: rotating_file
             path: "%env(APP_LOG_DIR)%/timing.%kernel.environment%.log"
-            level: debug
+            level: info
```

Comment:
This setting does not affect Datadog. It's only for local files. So we might keep it on `debug` however these logs might take a lot of disk space at some point 😁

Reply:
yes it was i understood i saw only INFO logs in datado :+1: